.ci/docker/common/install_onnx.sh

```diff
@@ -23,3 +23,18 @@ pip_install \
 
 # TODO: change this when onnx-script is on testPypi
 pip_install "onnx-script@git+https://github.com/microsoft/onnx-script@29241e15f5182be1384f1cf6ba203d7e2e125196"
+
+IMPORT_SCRIPT_FILENAME="/tmp/onnx_import_script.py"
```

Comment:
nit: how about moving this below the comment about caching, so the large caching comment goes before all the caching logic

Reply:
Sounds good